timing/mkAllTests

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
Is this something run by travis? Again, no comments explaining what this file is for.